src/plugins/plugin.gridlines.js

```diff
@@ -0,0 +1,166 @@
+'use strict';
+
+module.exports = function(Chart) {
+	var helpers = Chart.helpers;
```

Comment:
We changed the way how to require dependencies (see [this code](/chartjs/Chart.js/blob/master/src/plugins/plugin.filler.js#L9))